src/plugins/console/public/application/components/settings_modal.tsx

```diff
@@ -112,10 +128,12 @@ export function DevToolsSettingsModal(props: Props) {
     });
   }
 
-  const onIntervalChange: ChangeEventHandler<HTMLSelectElement> = useCallback(
-    (e) => setPollInterval(parseInt(e.target.value, 10)),
-    []
-  );
+  const onChange = useCallback((value: string) => {
```

Comment:
I think we should make this method a bit more descriptive:

```suggestion
  const onPollingIntervalChange = useCallback((value: string) => {
```